app/controllers/edit_merchant_details_controller.js

```diff
@@ -0,0 +1,89 @@
+'use strict'
+
+// NPM Dependencies
+const lodash = require('lodash')
+
+// Local Dependencies
+const responses = require('../utils/response')
+const paths = require('../paths')
+const serviceService = require('../services/service_service')
+const ukPostcode = require('uk-postcode')
+const countries = require('../services/countries.js')
+
+const merchantNameField = 'merchant-name'
```

Comment:
Generally we've been doing these sort of top-level `const` declarations in 'SCREAMING_SNAKE_CASE'.

Not sure this is a must.